firefox-ios/Client/Frontend/Browser/ClipboardBarDisplayHandler.swift

```diff
@@ -147,6 +105,9 @@ class ClipboardBarDisplayHandler: NSObject, URLChangeDelegate {
               shouldDisplayBar(url.absoluteString) else { return }
 
         lastDisplayedURL = url.absoluteString
-        delegate?.shouldDisplay(clipBoardURL: url)
+
+        AppEventQueue.wait(for: [.startupFlowComplete, .tabRestoration(windowUUID)]) { [weak self] in
```

Comment:
Turns out `.startupFlowComplete` was needed here?

Reply:
Although with testing, it seems `tabRestoration` sufficed because the start up flow seems to be complete at this point. I wanted to mimic how we handle deeplinks because essentially tapping on "GO" acts like a deeplink in that the browser is ready to handle showing the URL. Happy to remove if any concerns!